src/lephare/data_manager.py

```diff
@@ -43,33 +42,78 @@ def configure_directories(self):
-                now = datetime.datetime.now().strftime("%Y%m%dT%H%M%S")
-                run_directory = f"{this_os_cache}/runs/{now}"
-                print(
-                    f"""Lephare default working directory being created at
-                    {self.lephare_work_dir}."""
-                )
-                os.makedirs(run_directory)
-                os.symlink(run_directory, self.lephare_work_dir)
+            # get <default_cache> locations
+            default_os_cache = user_cache_dir("lephare", ensure_exists=True)
 
-                work_sub_directories = ["filt", "lib_bin", "lib_mag", "zphota"]
-                for sub_dir in work_sub_directories:
-                    os.makedirs(os.path.join(self.lephare_work_dir, sub_dir))
+            # create a runs directory in the default cache locations
+            os.makedirs(f"{default_os_cache}/runs", exist_ok=True)
+
+            # create a timestamped directory under runs
+            now = datetime.datetime.now().strftime("%Y%m%dT%H%M%S")
+            run_directory = f"{default_os_cache}/runs/{now}"
+            os.makedirs(run_directory, exist_ok=True)
+
+            # create the subdirectories in the new run directory
+            self.create_work_subdirectories(run_directory)
+
+            # create a symlink between <default_cache>/work and the timestamped directory
+            symlink_work_directory = f"{default_os_cache}/work"
+            # first remove the symlink if it already exists
+            if os.path.islink(symlink_work_directory):
+                os.unlink(symlink_work_directory)
+            os.symlink(run_directory, symlink_work_directory)
+
+            # set the LEPHAREWORK environment variable to the <default_cache>/work symlink
+            os.environ["LEPHAREWORK"] = symlink_work_directory
+
+            # set the instance variable to the <default_cache>/work symlink
+            self.lephare_work_dir = symlink_work_directory
         else:
+            # the lephare work dir env var is set, create subdirectories if needed
+            self.create_work_subdirectories(self.lephare_work_dir)
             print(
                 f"""User defined LEPHAREWORK is set. All intermediate files will
                 be written to {self.lephare_work_dir}."""
             )
+
+    def create_new_run(self):
+        """Create a timestamped directory to contain the output from the current run.
+        The newly created timestamped directory is symlinked to the path defined
+        by the LEPHAREWORK environment variable."""
+
+        lephare_work_dir = os.getenv("LEPHAREWORK", None)
+
+        # if the LEPHAREWORK environment variable is not a symlink,
+        # then this directory structure is unusual and we cannot create a new run.
+        # We'll raise an exception and direct the user to the documentation.
+        if not os.path.islink(f"{lephare_work_dir}"):
+            # TODO include link to documentation
```

Comment:
Captured this in #76 so we don't have to remember it in the back of our minds

Reply:
Thanks for doing that!